Postgres/AppDelegate.m

```diff
@@ -183,6 +206,14 @@ - (NSApplicationTerminateReply)applicationShouldTerminate:(NSApplication *)sende
     return NSTerminateLater;
 }
 
+- (BOOL)isDarkMode {
+    return [[[NSUserDefaults standardUserDefaults] valueForKey:kAppleInterfaceStyle] isEqualToString:kAppleInterfaceStyleDark];
```

Comment:
While it works in this case, you should in general  use `objectForKey:` instead of `valueForKey:`, because the latter is a KVC (key-value-coding) method. It would cause a problem if the string contained characters that have aspecial meaning in KVC, eg `.` or `@`. 

Also, for safety, don't use `isEqualToString:` on type `id`. Just use `isEqual:`. Otherwise someone could make Postgres.app crash by setting the default to a number.